util/all_tests.json

```diff
@@ -1,7 +1,8 @@
 [
   {
     "cmd": ["crypto/crypto_test"],
-    "valgrind_supp": ["valgrind_suppressions_crypto_test.supp"]
+    "valgrind_supp": ["valgrind_suppressions_crypto_test.supp"],
+    "shard": true
```

Comment:
The other `crypto/crypto_test` execution stanzas below should benefit from sharding as well(?)